doc/src/pair_ilj_cut.rst

```diff
@@ -0,0 +1,152 @@
+.. index:: pair_style ilj/cut
+
+pair_style ilj/cut command
+=============================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   pair_style ilj/cut cutoff
+
+* ilj/cut = name of the pair style
+* cutoff = global cutoff (distance units)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   pair_style ilj/cut 10.0
+   pair_coeff 1 1 4.0 7.0 6.0 3.5 0.0045
+
+Description
+"""""""""""
+
+.. versionadded:: 29Aug2024
+
+Pair style *ilj/cut* computes pairwise interactions from an Improved Lennard-Jones (ILJ) potential according to :ref:`(Pirani) <Pirani>`:
+
+
+.. math::
+
+   x = r/R_m   \\
+   n_x = \alpha*x^2 + \beta   \\
+   \gamma \equiv m  \\
+
+   V(x) = \epsilon \cdot \left( \frac{\gamma}{n_x - \gamma}  \left(\frac{1}{x} \right)^{n_x}  -  \frac{n_x}{n_x - \gamma}  \left(\frac{1}{x} \right)^{\gamma} \right)
+      \qquad r < r_c
+:math:`r_c` is the cutoff.
+
+
+An additional parameter, :math:`\alpha`, has been introduced in order to be able to recover the traditional LJ 12-6 with an adequate choice of parameters. With 
+:math:`R_m \equiv r_0 = \sigma \cdot 2^{1/6}`, :math:`\alpha = 0`, :math:`\beta = 12` and :math:`\gamma = 6` it is straightforward
+to prove that LJ 12-6 is obtained.
+
+
+This potential provides some advantages with respect to the Lennard-Jones (LJ) potential and can be really useful for molecular dynamics simulations, as one can see from :ref:`(Pirani) <Pirani>`.
+It can be used for neutral-neutral (:math:`\gamma = 6`), ion-neutral (:math:`\gamma = 4`) or ion-ion systems (:math:`\gamma = 1`). 
+It removes most of the issues at short- and long-range of the LJ model.
+
+
+It is possible to verify that using (:math:`\alpha= 4`), (:math:`\beta= 6`) and (:math:`\gamma = 6`), at the equilibrium distance, 
+the first and second derivatives of ILJ coincide with those of LJ 12-6 ( and the reduced force constant amounts to the typical 72). 
+In this case, LJ provides a long-range coefficient with a factor of 2 compared with the ILJ. Also, the short-range interaction is overestimated by LJ. 
+The ILJ potential solves both problems.
+
+
+The analysis of a diverse amount of systems verified that (:math:`\alpha= 4`) works very well. 
+In some special cases (e.g. those involving very small multiple charged ions) this factor may take a slightly different value. 
+The parameter (:math:`\beta`) codifies the hardness (polarizability) of the interacting partners, and for neutral-neutral systems it ranges from 6 to 11. 
+Moreover, the modulation of (:math:`\beta`) permits to indirectly consider the role of further interaction components (such as the charge transfer int he perturbative limit) 
+and mitigates the effect of some uncertainty in the data.
+
+
+A couple of examples of the applications of the ILJ potential should be useful to show its wide variety of uses:
```

Comment:
The manual is just documenting the functionality, not a room for "commercials". People should be able to figure out applications of the model from looking at citations of the paper describing it.